src/main/java/com/nyxei/pollingus/impl/WhatsappBusinessCloudApi.java

```diff
@@ -0,0 +1,45 @@
+package com.nyxei.pollingus.impl;
+
+
+
+import static com.nyxei.pollingus.WhatsappApiServiceGenerator.createService;
+
+
+
+import com.nyxei.pollingus.domain.messages.Message;
+import com.nyxei.pollingus.domain.messages.response.MessageResponse;
+import com.nyxei.pollingus.service.WhatsappBusinessCloudApiService;
+
+
+import static com.nyxei.pollingus.WhatsappApiServiceGenerator.executeSync;
+
+/**
+ * Implementation of WhatsApp Business Platform Cloud API with synchronous/blocking
+ * method calls.
+ *
+ * @see <a href="https://developers.facebook.com/docs/whatsapp/cloud-api">WhatsApp Business Platform Cloud API</a>
+ */
+public class WhatsappBusinessCloudApi {
+
+    private final WhatsappBusinessCloudApiService whatsappBusinessCloudApiService;
+
+    public WhatsappBusinessCloudApi(String token) {
+        this.whatsappBusinessCloudApiService = createService(WhatsappBusinessCloudApiService.class, token);
```

Comment:
You could avoid making `WhatsappBusinessCloudApi` final by making it singleton in the container.